operator/validator/controller.go

```diff
@@ -864,6 +864,9 @@ func (c *controller) onShareStop(pubKey spectypes.ValidatorPK) {
 	if ok {
 		vc.RemoveShare(v.Share.Share.ValidatorIndex)
 		if len(vc.Shares) == 0 {
+			for _, slotQueue := range vc.Queues {
+				slotQueue.StopQueueF()
+			}
```

Comment:
you can't do this here, accessing Queues requires mutex acquiring. best is to have a function inside the committee struct that handles this.